lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,17 +552,25 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.warning(`The cloud instance discovery network request's status code is: ${response.status}`);
+                this.logger.warning("metadata is an empty list.");
+                
+                const tenantDiscoveryEnpoint = response.body.tenant_discovery_endpoint;
```

Comment:
Possible null ref, response can be null, body can be null...

Reply:
How could these be null? This is in a try/catch, so if the network request fails, it will never get to this point. A status and body will always be returned for any completed network request, per HttpClient.ts.